.github/actions/setup-python-env/action.yaml

```diff
@@ -5,7 +5,7 @@ inputs:
   python-version:
     description: "Version of Python to use for testing"
     required: false
-    default: '3.8'
+    default: "3.8"
```

Comment:
Shall we default this to most recent, rather than least recent? I could see either perspective.

Reply:
Figuring that using an older version would be better to catch uses of newer language features.